src/domain/usecases/CopyInUserUseCase.ts

```diff
@@ -0,0 +1,74 @@
+import _ from "lodash";
+
+import { Future, FutureData } from "../entities/Future";
+import { User } from "../entities/User";
+import { AccessElements, UpdateStrategy, AccessElementsKeys, UserRepository } from "../repositories/UserRepository";
+import { Id } from "../entities/Ref";
+
+export class CopyInUserUseCase {
+    constructor(private userRepository: UserRepository) {}
+
+    public execute(options: CopyInUserOptions): FutureData<void> {
+        return this.getUsersToUpdate(options.selectedUsersIds).flatMap(users => {
+            const usersBatches = _.chunk(users, 50);
+
+            const $requests = usersBatches.map(usersToUpdate => {
+                const usersToSave = this.applyCopyToUsers(usersToUpdate, options);
+                return this.saveUsers(usersToSave);
+            });
+
+            // Update batch of 10 users
```

Comment:
Update comment according the new batch size (or maybe remove it since the `userBatches` is very easy to understand)